File: 1436/main.cpp
#include <iostream>
#include <string>

using namespace std;

int main(void) {

	int n ;
	cin >> n;

	int count = 0;
	int start = 666;

	while (count < n) {
		string str = to_string(start);
	
		if (str.find("666") != -1) {
			count++;
		}

		start++;
	}

	cout << start - 1 << endl;
}

File: 1436/Main.java
import java.util.Scanner;

public class Main {
	public static void main(String [] args) {
		
		Scanner sc = new Scanner(System.in);

		int n = sc.nextInt();

		int count = 0;
		int start = 666;

		while ( count < n ) {

			String str = Integer.toString(start);

			if ( str.find("666") == true ) {
				count += 1;
			}

			start += 1;
		}

		System.out.print(start - 1);
	}
}
